CODEOWNERS

```diff
@@ -0,0 +1 @@
+*       @serratus @sgeisbacher
```

Comment:
That means that both of us will be added as reviewers on every single PR (according to https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)
> Code owners are automatically requested for review when someone opens a pull request that modifies code that they own. 

Let's start there and refine over time 👍

Reply:
yeah, thats how i understand it 😅